example/lib/payments/widget/payments_body_widgets.dart

```diff
@@ -0,0 +1,83 @@
+part of 'payments_page.dart';
+
+
```

Comment:
Excess new line

Reply:
Done in https://github.com/getkevin/kevin-flutter/pull/20/commits/205dce361bc77742a6c6fc88391c686ae645f87a